Mutation testing (#12)
moonwell-fi/mtoken-fixes: README.md

certoraRun certora/confs/MErc20DelegateFixer.conf
```

## Mutation Testing

Use certora gambit to generate mutaions for `MErc20DelegateFixer` and then run each mutation against unit, integration tests and formal specification using `runMutation` script. The script generates a `Result.txt` file which stores following details for each mutations:
- mutant diff with original contract
- unit/integration test results with number and list of failing tests if any
- result of certora formal verification against mutant with details such as number of failed rules, their list and certora prover cli job url

Finally it logs total number of failed mutations.

> Following steps needs to be followed for mutation testing:

Reviewer: ```suggestion
Follow these steps to run the mutation test scripts:
```
